frame/elections/src/lib.rs

```diff
@@ -236,16 +236,16 @@ decl_storage! {
 		// [`all_approvals_of`]. Furthermore, each vector of scalars is chunked with the cap of
 		// `APPROVAL_SET_SIZE`.
 		pub ApprovalsOf get(fn approvals_of):
-			map hasher(blake2_256) (T::AccountId, SetIndex) => Vec<ApprovalFlag>;
+			map hasher(twox_64_concat) (T::AccountId, SetIndex) => Vec<ApprovalFlag>;
 		/// The vote index and list slot that the candidate `who` was registered or `None` if they
 		/// are not currently registered.
 		pub RegisterInfoOf get(fn candidate_reg_info):
-			map hasher(blake2_256) T::AccountId => Option<(VoteIndex, u32)>;
+			map hasher(blake2_128_concat) T::AccountId => Option<(VoteIndex, u32)>;
```

Comment:
```suggestion
			map hasher(twox_64_concat) T::AccountId => Option<(VoteIndex, u32)>;
```

As far as I can see, this key is set with `submit_candidacy`, and we insert `let who = ensure_signed(origin)`.

So safety by transaction here.